src/gen_random_network.py

```diff
@@ -4,20 +4,20 @@
 import numpy as np
 
 import util
+from handl import regularized_laplacian, rkhs_factor
 
 from collections import defaultdict
 from sklearn.externals import joblib
 from sklearn.externals.joblib import Parallel, delayed
 from i_o import get_logger
+from time import time
 
 
 def parse_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument('-s', '--species_name', type=str, required=True)
-    parser.add_argument('-n', '--n_graphs', type=int, required=True)
     parser.add_argument('-e', '--edgelist', type=str, required=True)
-    parser.add_argument('-d', '--output_dir', type=str, required=True)
-    parser.add_argument('-j', '--n_jobs', type=int, required=False, default=1)
+    parser.add_argument('-o', '--output', type=str, required=True)
+    parser.add_argument('-nt', '--n_tries', type=int, required=False, default=50)
```

Comment:
Not sure if I understand why you do `n_tries`, instead of redoing any failed ones and setting `max_tries`.